### ISSUE TITLE
Remove default values like stroke-dasharray="none" stroke-opacity="1"

### ISSUE THREAD

**Issue by atom3001** [svg/svgo#1973]
Remove default values like stroke-dasharray="none" stroke-opacity="1"

**Comment by KTibow** [comment]
Tip: SVGOMG hasn't been updated for a while, and won't be updated until #1943 is merged. https://omfg.svg.beauty/ is slightly more up to date, but it's best to directly use the library/CLI if you can.

Commentary on the issue: This is already done to some degree, but I wouldn't be surprised if it doesn't work in all cases. Want to share an example SVG where it should remove it but doesn't?

**Comment by KTibow** [comment]
It seems the optimization is disabled when the path is linked in a `use`. I haven't found where this optimization is applied yet so I don't have much to add.

**Comment by atom3001** [comment]
Yes, if I drop the <use> elements the path styles get cleaned properly. good spot, thanks. So I could cleanup before cloning. Shame to have a fragmented workflow. But writing plugs to do a second cleaning pass is beyond me for now. Thanks again for looking

**Comment by KTibow** [comment]
I checked and the relevant plugin is `removeUnknownsAndDefaults`, which actually doesn't clean defaults on attributes with an `id`

**Comment by atom3001** [comment]
Interesting, thanks for digging deeper @KTibow. Still leaves me in the same boat. Perhaps it would be possible to disable that id  limitation?
